### PR TITLE
Fix getting swagger doc on travis

### PR DESCRIPTION
**Pull request by rkondratowicz**
Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: publish-swagger-docs.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ then
   exit 0
 fi
 
+./gradlew clean installDist
```
Comment (timja-kainos) [inline publish-swagger-docs.sh]: shouldn't this be done in the travis script section?
https://github.com/hmcts/draft-store/blob/master/.travis.yml#L24

Comment (rkondratowicz) [reply publish-swagger-docs.sh]: imho, it feel more 'self contained' when it's here

**Comment by doncem** [comment]
I vote for moving to `.travis.yml` as well as precondition. Final result will be just a step of after success:
```yml
after_success:
  - codacy publish
  - test $TRAVIS_BRANCH = "master" && ./gradlew clean installDist && ./publish-swagger-docs.sh
```

**Comment by rkondratowicz** [comment]
how about we merge it as is and see if it fixes the issue and then refactor to using 'after-success' in travis file?

**Pull request merged**